docs/src/main/sphinx/release/release-393.md

```diff
@@ -0,0 +1,116 @@
+# Release 393 (10 August 2022)
+
+## General
+
+* Add support for the SQL [Merge](/sql/merge) statement. ({issue}`7933`)
+
+## Security
+
+* Fix potential incorrect rejection when using OAuth 2.0 refresh tokens. ({issue}`13168`)
+* Fix showing outdated OAuth 2.0 refresh tokens in the web UI. ({issue}`13168`)
+
+## Docker image
+
+* Add image for `ppc64le`. ({issue}`13522`)
+
+## ClickHouse connector
+
+* Add support for the `unsupported-type-handling` catalog configuration
+  property. ({issue}`13542`)
+* Improve performance for queries with selective joins. ({issue}`13334`)
```

Comment:
Q: Dynamic filtering is something people now understand and have an idea of what it does. Should we mention it by name here as well?

Reply:
Does the end user actually want to know that it's dynamic filtering? I think we generally want to frame it in context of how it matters to users, so whether the performance comes from x strategy or y strategy is almost besides the point, and if they're curious, they can click on the issue to figure out what's happening.

Reply:
That's a fair point. I was thinking more along "marketing" lines (which should probably be kept out of release notes and left to the blog posts).

Reply:
Yeah, if dynamic filtering is cool and we want to drum up hype for it, let's get a blog post about it written. I'll follow up with @raunaqmorarka on that.